### PR TITLE
Support streamlit bot interactive mode

### PR DESCRIPTION
**Pull request by uni-zhuan**

### CODE REVIEW

**Comment by uni-zhuan** [comment]
use `poetry run yival bot experiment_path` to get the bot up and running on the web

current display:
image
image

**Review by crazycth — COMMENTED** [review]
Do you need to change .toml?

File: demo/tutorial_notebook/animal_story.ipynb [hunk 1]
```diff
@@ -59,12 +59,12 @@
...
     "\n",
     "```\n",
-    "custom_function: demo.logo_generation.logo_generation\n",
+    "custom_function: demo.animal_story.image_generation\n",
     "description: Generated experiment config\n",
     "dataset:\n",
     "  data_generators:\n",
     "    openai_prompt_data_generator:\n",
-    "      chunk_size: 100000\n",
+    "      chunk_size: 2\n",
```
Comment (crazycth) [inline demo/tutorial_notebook/animal_story.ipynb:67]: Leave the chunk_size = 10000 , it doesn't matter

Comment (uni-zhuan) [reply demo/tutorial_notebook/animal_story.ipynb]: solved

**Comment by uni-zhuan** [comment]
> Do you need to change .toml

**Review by crazycth — APPROVED** [review]

**Pull request merged**